src/uct/ib/base/ib_md.c

```diff
@@ -1385,11 +1387,12 @@ ucs_status_t
 uct_ib_md_open(const char *md_name, const uct_md_config_t *uct_md_config, uct_md_h *md_p)
 {
     const uct_ib_md_config_t *md_config = ucs_derived_of(uct_md_config, uct_ib_md_config_t);
+    uct_ib_md_t *md = NULL;
```

Comment:
need to init status to UNSUPPORTED and check it (in addition to assertion on md)